### PR TITLE
Refs #23938 - UI table API call for subtasks action name

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

File: webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ export const SubTasksPage = props => {
       { caption: __('Sub tasks') },
     ],
   });
-  const createHeader = actionName => actionName ? sprintf(__('Sub tasks of %s'), actionName) : __('Sub tasks');
+  const createHeader = actionName =>
+    actionName ? sprintf(__('Sub tasks of %s'), actionName) : __('Sub tasks');
```
Comment (adamruzicka) [inline webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js:19]: Is this needed? From what I've seen the breadcrumbs are always `Tasks > $action_name > Sub tasks`

Comment (MariaAga) [reply webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js:19]: In some tasks the action is empty. But we can do a fall back like we did here instead: https://github.com/adamruzicka/foreman-tasks/blob/c6bf988bb25fc473d44d4ee8ca6d18353a168f6c/webpack/ForemanTasks/Components/TasksTable/TasksTableSelectors.js#L39

Comment (adamruzicka) [reply webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js:19]: I see, then it is fine the way it is

**Comment by adamruzicka** [comment]
Thank you @MariaAga !

**Pull request merged**